app/main.py

```diff
@@ -9,8 +9,8 @@ def __init__(self, name: str, age: str) -> None:
 
 def create_person_list(people: list) -> list:
     person_list = [Person(person["name"], person["age"]) for person in people]
-    for prsn in people:
-        if prsn.get("wife"):
-            Person.people[prsn["name"]].wife = Person.people[prsn["wife"]]
-            Person.people[prsn["wife"]].husband = Person.people[prsn["name"]]
+    for man in people:
```

Comment:
I think man it's not a good name for this variable))
Because it could be a woman also))

`for person in people` will be ok)

Reply:
man. Not men :D 
Anyway changed